### PR TITLE
Simple Docs for Atomic Swap Pallet

### PR DESCRIPTION
**Pull request by danforbes**
Docs re https://github.com/paritytech/substrate/pull/6349

### CODE REVIEW

File: frame/atomic-swap/src/lib.rs [hunk 1]
```diff
@@ -15,7 +15,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! # Atomic swap support pallet
+//! # Atomic Swap
+//!
+//! A module for atomically sending funds.
+//!
+//! - [`scheduler::Trait`](./trait.Trait.html)
```
Comment (shawntabrizi) [inline frame/atomic-swap/src/lib.rs:22]:
```suggestion
//! - [`atomic-swap::Trait`](./trait.Trait.html)
```

Comment (shawntabrizi) [reply frame/atomic-swap/src/lib.rs:22]: Might be underscore, i dunno

Comment (danforbes) [reply frame/atomic-swap/src/lib.rs]: Gah, thank you...believe it is an underscore.

**Comment by danforbes** [comment]
Correct, this is strictly for the Rustdocs.

**Pull request merged**